src/cli.ts

```diff
@@ -53,14 +52,18 @@ $ pwa-asset-generator --help
     --log false
 `,
   {
-    flags,
+    // TODO: remove when inferred meow types are corrected
+    // Default vars can be number and boolean too but declared to be string in type defs
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore
+    flags: constants.FLAGS,
   },
 );
 const logger = preLogger('cli', cli.flags);
 
-(async () => {
+(async (): Promise<void> => {
```

Comment:
It is not necessary to add return types for anonymous functions.

Reply:
Project uses @typescript-eslint eslint settings which recommends explicit-function-return-type and warns when it's not there. That's the main reason of return types everywhere.